nengo_viz/templates/page.html

```diff
@@ -10,7 +10,10 @@
     <script src="static/viz_slider.js"></script>
     <link href="static/viz_slider.css" rel="stylesheet" type="text/css" />
     <script src="static/viz_sim_control.js"></script>
-    <link href="static/viz_sim_control.css" rel="stylesheet" type="text/css" />    
+    <link href="static/viz_sim_control.css" rel="stylesheet" type="text/css" /> 
+    <script src="static/lib/js/jquery-2.1.3.min.js"></script>
+    <script src="static/lib/js/bootstrap.min.js"></script>
+    <link href="static//lib/css/bootstrap.min.css" rel="stylesheet" type="text/css"> 
```

Comment:
We should have some consistency in the locations of these files.  If you think there should be this separate ```lib``` directory, then interact-1.2.2.js and d3.v3.min.js should go in there too.  

(note: to move files correctly with git, do something like ```git mv interact-1.2.2.js lib/js/interact-1.2.2.js```)